### ISSUE TITLE
[Bug]: XP Obelisk from EnderIO is not working to store XP.

### ISSUE THREAD

**Issue by sS3CKk** [AllTheMods/ATM-10#1524]
### Possible Fixes

Yes

### Modpack Version

ATM10 2.10

### What happened?

XP Obelisk from EnderIO is not working when you click to store XP, its just working the retrieve XP.

**Comment by alekthefirst** [comment]
seems to work fine?
https://github.com/user-attachments/assets/f000a483-f511-45b0-bce6-2a6d6bd8ce16

**Comment by Silicat-git** [comment]
Having the same issue, seems to be fluids conflicting with each other.
I filled the obelisk with xp made from productive bees through a dimensional tank and fluid pipes

This results in "XP Fluid" beeing in the obelisk and beeing unable to deposit levels.
If I empty the obelisk and deposit all levels (works now) the fluid inside the obelisk is "XP Juice" which you can deposit levels to.

**Comment by elowanut** [comment]
Yeah, the xp ‘fluids’ aren’t unified, there’s a couple variants. You can, however, use the ender xp juice to fuel things like the hero’s medallion via piping.

**Comment by polyjitter** [comment]
Hey, the caution warning has been lifted for Ender IO 7.1.x., and in the newest version this has been resolved. Any chance we'll see an update to include it soon? I really want to be able to just have players install the modpack and not have to mess with content management :)

https://www.curseforge.com/minecraft/mc-mods/ender-io/files/6065326

> We're now lifting the caution warning for 7.1.x, as we've seen no major bug reports since it was released. Thanks to everyone who has helped to test these updates.
